base/linalg/bidiag.jl

```diff
@@ -13,23 +13,41 @@ type Bidiagonal{T} <: AbstractMatrix{T}
     end
 end
 """
-    Bidiagonal(dv, ev, isupper)
+    Bidiagonal(dv, ev, isupper::Bool)
 
 Constructs an upper (`isupper=true`) or lower (`isupper=false`) bidiagonal matrix using the
 given diagonal (`dv`) and off-diagonal (`ev`) vectors.  The result is of type `Bidiagonal`
 and provides efficient specialized linear solvers, but may be converted into a regular
 matrix with [`full`](:func:`full`). `ev`'s length must be one less than the length of `dv`.
+
+**Example**
+
+```julia
+d = rand(5)
+e = rand(4)
```

Comment:
would use `dv` and `ev` as variable names to avoid overwriting the usual MathConst `e`